src/kvirc/kvs/KviKvsTimerManager.cpp

```diff
@@ -202,7 +202,7 @@ void KviKvsTimerManager::timerEvent(QTimerEvent *e)
 	KviKvsTimer * t = m_pTimerDictById->find(iId);
 	if(!t)
 	{
-		qDebug("Internal error: got an nonexistant timer event");
+		qDebug("Internal error: Received a non existent timer event");
```

Comment:
Isn't the "R" in "received" supposed to be lowercase as the "t" in "trying" in this same commit?